core/utils/backoff_ticker_test.go

```diff
@@ -0,0 +1,46 @@
+package utils_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+// assertTick asserts for a tick during the waitDuration
+func assertTick(t *testing.T, ticker *utils.BackoffTicker, doesTick bool, waitDuration time.Duration) {
+	ticked := false
+
+	waitCh := time.After(waitDuration)
+
+	select {
+	case <-ticker.Ticks():
+		ticked = true
+	case <-waitCh:
+	}
+
+	assert.Equal(t, doesTick, ticked)
+}
+
+func TestBackoffTicker(t *testing.T) {
+	t.Parallel()
```

Comment:
To avoid flakiness with a test with lots of sub-second timing expectations, I'd recommend de-parallelizing it. Even then, I think we should keep an eye on this one, and consider increasing all of the durations by an order of magnitude (without changing the fundamental expectations). My guiding rule of thumb: CI boxes are shit

Reply:
The variance here is constant so I don't believe that we need to increase the assert waitDuration with each tick. However I can increase the base variance just to be sure